spec/lib/dfe_sign_in_api_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+require 'dfe_sign_in_api'
+
+RSpec.describe DFESignIn::API do
+  describe '#users' do
+    it 'returns the users from the API ' do
+      stub_api_response_for_page(1)
+      response = described_class.new.users(page: 1)
```

Comment:
Since the default value of the page argument is 1, could we rewrite `response = described_class.new.users(page: 1)` to be `response = described_class.new.users`?
This way this current test describes making a successful API call while the other test below is testing making a successful API call for a **specific page**.